mm2src/adex_cli/src/adex_proc/smart_fraction_fmt.rs

```diff
@@ -0,0 +1,97 @@
+use std::cell::Cell;
+use ToString;
```

Comment:
Clippy suggests to remove this and other suggestions. I think we should start setting up CI checks for adex-cli @ozkanonur, what do you think? it can be done in another PR on this PR but we need to merge it before this PR to make sure that the cli code follows the conventions.

Reply:
there it is https://github.com/KomodoPlatform/atomicDEX-API/pull/1818

Reply:
Thank you @ozkanonur

Reply:
done